staging/podsecuritypolicy/rbac/README.md

```diff
@@ -129,9 +131,9 @@ We can then create role bindings to grant those permissions.
 
 In this example, we will create `ClusterRoleBindings` to grant the roles to groups cluster-wide.
 
-1. `privileged`: this group is bound to the `privilegedPSP` role and `restrictedPSP` role which gives users
+1. `privileged-psp-user`: this group is bound to the `privileged-psp-user` role and `restricted-psp-user` role which gives users
 in this group access to both policies.
-1. `restricted`: this group is bound to the `restrictedPSP` role.
+1. `restricted-psp-user`: this group is bound to the `restricted-psp-user` role.
 1. `system:authenticated`: this is a system group for any authenticated user.  It is bound to the `edit`
 role which is already provided by the cluster.
```

Comment:
> `system:authenticated`: this is a system group for any authenticated user.  It is bound to the `edit`
 role which is already provided by the cluster.

Does it mean that any authenticated user has `edit` role?
Why do we bound PSPs to the `edit` role and not to the `system:authenticated` directly?

Reply:
Is this so all the example users will be able to create pods? Maybe just bind a "developers" group directly to the "edit" role.

```
kubectl create -f=example-pod.yaml --as=jane --as-group=developers --as-group=restricted
kubectl create -f=example-pod.yaml --as=john --as-group=developers --as-group=privileged
```

Binding `edit` to `system:authenticated` seems like bad practice, even in an example.